Source/Scene/PointCloud3DTileContent.js

```diff
@@ -1180,16 +1277,76 @@ define([
 
     var scratchComputedTranslation = new Cartesian4();
     var scratchComputedMatrixIn2D = new Matrix4();
+    var scratchModelMatrix = new Matrix4();
+
+    function decodeDraco(content, context) {
+        if (content._decodingState === DecodingState.READY) {
+            return false;
+        }
+        if (content._decodingState === DecodingState.NEEDS_DECODE) {
+            var parsedContent = content._parsedContent;
+            var draco = parsedContent.draco;
+            var decodePromise = DracoLoader.decodePointCloud(draco, context);
+            if (defined(decodePromise)) {
+                content._decodingState = DecodingState.DECODING;
+                decodePromise.then(function(result) {
+                    content._decodingState = DecodingState.READY;
+                    var decodedPositions = defined(result.POSITION) ? result.POSITION.array : undefined;
+                    var decodedRgb = defined(result.RGB) ? result.RGB.array : undefined;
+                    var decodedRgba = defined(result.RGBA) ? result.RGBA.array : undefined;
+                    var decodedNormals = defined(result.NORMAL) ? result.NORMAL.array : undefined;
+                    var decodedBatchIds = defined(result.BATCH_ID) ? result.BATCH_ID.array : undefined;
+                    if (defined(decodedPositions) && content._isQuantizedDraco) {
+                        var quantization = result.POSITION.data.quantization;
+                        var scale = quantization.range / (1 << quantization.quantizationBits);
+                        content._quantizedVolumeScale = Cartesian3.fromElements(scale, scale, scale);
+                        content._quantizedVolumeOffset = Cartesian3.unpack(quantization.minValues);
+                        content._quantizedRange = (1 << quantization.quantizationBits) - 1.0;
```

Comment:
`._quantizedRange` !== `quantization.range`? Maybe clarify terminology here.

Reply:
Yeah this is kind of confusing. I'll add a comment to the code in https://github.com/AnalyticalGraphicsInc/cesium/pull/6721 because the behavior changed slightly:
https://github.com/AnalyticalGraphicsInc/cesium/pull/6721/files#diff-e12a976367111c8fb95abea65e0b6ab0R1228

Reply:
Comment added here: https://github.com/AnalyticalGraphicsInc/cesium/pull/6721/commits/6161a9615e41c969400a079e4dd379a1d265c79e